mapie/multi_label_classification.py

```diff
@@ -781,13 +824,31 @@ def predict(
             len(alpha_np),
             axis=2
         )
-        self.r_hat, self.r_hat_plus = self._get_r_hat_plus(
-            bound,
-            delta,
-        )
-        self.lambdas_star = self._find_lambda_star(self.r_hat_plus, alpha_np)
-        y_pred_proba_array = (
-            y_pred_proba_array >
-            self.lambdas_star[np.newaxis, np.newaxis, :]
-        )
+        if self.metric_control == 'precision':
+            self.n_obs = len(self.risks)
+            self.r_hat = self.risks.mean(axis=0)
+            self.n_obs = len(self.risks)
+            self.valid_index, self.p_values = _ltt_procedure(self.r_hat,
+                                                             alpha_np,
+                                                             delta,
+                                                             self.n_obs)
```

Comment:
STYLE: add newlines as done in lines836-840)

Reply:
Done